website/docs/CBD.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: cbd
+---
+
+# Condition Based Decryption
+
+Condition Based Decryption (CBD) is a programmable access control service, in which decryption rights are based on the verified fulfillment of predefined conditions.
+Access conditions can be EVM-based (e.g. does the designated recipient own this NFT), RPC-driven (e.g. did the designated recipient commence active mining before this block) or time-based (e.g. has a preset period of time elapsed, after which the recipient's requests will be ignored).
+These conditions are also composable and can be combined in any logical sequence or decision tree.  
+
+CBD involves splitting the secret (a decryption key) into multiples _shares_ and distributing those among authorized and collateralized node operators ([Stakers](https://threshold.network/earn/staker) in the Threshold network).
```

Comment:
```suggestion
CBD involves splitting a joint secret (a decryption key) into multiples _shares_ and distributing those among authorized and collateralized node operators ([Stakers](https://threshold.network/earn/staker) in the Threshold network).
```

Also, "joint" seems to be more in line with the new codename...

Reply:
:joy: